models/en-US.json

```diff
@@ -478,6 +483,24 @@
                 {
                     "name": "AMAZON.ScrollUpIntent",
                     "samples": []
+                },
+                {
+                    "name": "ChangeStatusIntent",
+                    "slots": [
+                        {
+                            "name": "statusmessage",
+                            "type": "AMAZON.SearchQuery",
+                            "samples": [
+                                "{statusmessage}"
```

Comment:
Try adding 
"change status to {statusmessage}",
"set status to {statusmessage}"

these to the slot elicitation samples as well.

The user once he is asked "Alright, what's the new status?"

He can reply with "set status to {searchquery}"

Currently it is including "set status to" as well.

Check with these two slot utterances if everything works fine,
if yes, add them as well.